STUDIO/org.openl.rules.workspace/src/org/openl/rules/project/abstraction/RulesProject.java

```diff
@@ -35,6 +40,8 @@ public class RulesProject extends UserWorkspaceProject {
     private Repository designRepository;
     private String designFolderName;
     private final LockEngine lockEngine;
+    private static final String TAGS_FILE_NAME = "tags.properties";
```

Comment:
Static constants should be placed at the top level of the class, as per Java code conventions.